marketplace/commercetools-products/src/dialog.js

```diff
@@ -0,0 +1,91 @@
+/* global CTPicker */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from '@contentful/forma-36-react-components';
+
+export class CommerceToolsDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.pickerContainerRef = React.createRef();
+    this.picker = null;
+
+    this.onCancel = this.onCancel.bind(this);
+    this.onInsert = this.onInsert.bind(this);
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <div ref={this.pickerContainerRef} className="picker-container" />
+
+        <div className="picker-controls">
+          <Button onClick={this.onInsert} buttonType="positive" style={{ marginRight: 16 }}>
```

Comment:
Avoid styles definitions as inline object literals. Extract them to constant

```
const styles = {
  pickerButton: { marginRight: 16 },
  somethingElse: { color: 'red' }
}
```

Potentially in a separate file.

Consider using https://github.com/emotion-js/emotion if you see a benefit (utility vs bundle size addition).